app/models/actor.rb

```diff
@@ -61,11 +61,6 @@ class Actor < ApplicationRecord
             uniqueness: {
               case_sensitive: false,
               allow_nil: true
-            },
-            format: {
-              with: /\A\d{15,15}[0-9X]{1,1}\z/,
-              message: 'must be valid',
-              allow_nil: true
```

Comment:
Are we now just going to assume all ORCIDs in the system are valid? I guess even in the case where we are adding an actor in the create/edit work form, we are going out to the ORCID API to validate that ORCID, so we could safely assume it is in the correct format?

Reply:
@nicktiberi I can go either way on this one. My thoughts on validation, is that they are something for user-submitted data, and produce an error message for users to fix their issues. Since the orcid is now coming from PSU Identity, I think we can make the case that this validation can go away. BUT, I would like to see our code doing some type of checks and/or formatting of what comes back from the PSU Identity service, to make sure that all of our orcids are stored in a canonical way. It's possible that how PSU Identity stores orcids and how we want to store them are different